src/xdocs/config_coding.xml

```diff
@@ -464,14 +464,58 @@ public class A {
           more readable if it comes after the last <code>case</code>.
         </p>
       </subsection>
-
+      <subsection name="Properties">
+        <table>
+          <tr>
+            <th>name</th>
+            <th>description</th>
+            <th>type</th>
+            <th>default value</th>
+          </tr>
+          <tr>
+            <td>skipIfLastAndSharedWithCase</td>
+            <td>whether to allow <code>default</code> along with case if they are not last</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td>false</td>
+          </tr>
+        </table>
+      </subsection>
       <subsection name="Examples">
         <p>
           To configure the check:
         </p>
         <source>
 &lt;module name=&quot;DefaultComesLast&quot;/&gt;
         </source>
+        <p>
+          or
```

Comment:
Check with option on isn't part of the default check. It should be its own sentence, not a continuation of the last.
Change this to: `To configure the check for `...